cmd/tsbs_generate_data/main.go

```diff
@@ -305,10 +317,10 @@ func getSerializer(sim common.Simulator, format string, out *bufio.Writer) seria
 		out.WriteString("\n")
 
 		return &serialize.TimescaleDBSerializer{}
-	default:
-		fatal("unknown format: '%s'", format)
-		return nil
 	}
+
+	fatal("unknown format: '%s'", format)
```

Comment:
would rather keep as a `default`

Reply:
In this particular case - yes, but still, in general having no clear return value at the end of the function is not a good approach.

Reply:
Updated